sklearn/ensemble/_forest.py

```diff
@@ -673,8 +673,8 @@ def predict_proba(self, X):
 
         if len(all_proba) == 1:
             return all_proba[0]
-        else:
-            return all_proba
```

Comment:
I'm vaguely OK with the rest, but I typically think that this particular change hurts readability.

It's worth removing the else clause when you have a `if foo: return bar` at the very beginning, but when it's at the end like that, I prefer keeping the if/else structure.